ruby/rails/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.3.1
```

Comment:
We will move to a more compact and optimized Alpine image later. For now let's stick to the default one.